impexp-client/build.gradle

```diff
@@ -141,6 +141,21 @@ distributions.main {
     }
 }
 
+distributions {
+    minimized {

```

Comment:
Not really an issue, but I suggest to only have one `distributions` property which then contains both `main` and `minimized`:

```gradle
distributions {
  main {
    ...
  }

  minimized {
    ...
  }
}
```